content/en/docs/ops/configuration/mesh/app-health-check/index.md

```diff
@@ -106,84 +106,21 @@ This approach rewrites the application `PodSpec` readiness/liveness probe, such
 request to application, and strips the response body only returning the response code.
 
-This feature is enabled when installing with the `default` profile. If you find that the profile used to install Istio does not have it enabled, you have two ways to enable the rewrite of the liveness HTTP probes:
+This feature is enabled by default when installing with any of our [profiles](/docs/setup/additional-setup/config-profiles/). This option is recommended because it requires no code change in your application.
 
-#### Enable globally via install option
-
-[Install Istio](/docs/setup/install/istioctl/) with `--set values.sidecarInjectorWebhook.rewriteAppHTTPProbe=true`.
-
-**Alternatively**, update the configuration map of Istio sidecar injection:
-
-{{< text bash >}}
-$ kubectl get cm istio-sidecar-injector -n istio-system -o yaml | sed -e 's/"rewriteAppHTTPProbe": false/"rewriteAppHTTPProbe": true/' | kubectl apply -f -
-{{< /text >}}
-
-The above installation option and configuration map, each instruct the sidecar injection process to automatically
-rewrite the Kubernetes pod's spec, so health checks are able to work under mutual TLS. No need to update your app or pod
-spec by yourself.
-
-{{< warning >}}
-The configuration changes above (by install or by the configuration map) effect all Istio app deployments.
-{{< /warning >}}
-
-#### Use annotations on pod
-
-<!-- Add samples YAML or kubectl patch? -->
-
-Rather than install Istio with different options, you can [annotate the pod](/docs/reference/config/annotations/) with `sidecar.istio.io/rewriteAppHTTPProbers: "true"`. Make sure you add the annotation to the [pod resource](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/) because it will be ignored anywhere else (for example, on an enclosing deployment resource).
-
-{{< text yaml >}}
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: liveness-http
-spec:
-  selector:
-    matchLabels:
-      app: liveness-http
-      version: v1
-  template:
-    metadata:
-      labels:
-        app: liveness-http
-        version: v1
-      annotations:
-        sidecar.istio.io/rewriteAppHTTPProbers: "true"
-    spec:
-      containers:
-      - name: liveness-http
-        image: docker.io/istio/health:example
-        ports:
-        - containerPort: 8001
-        livenessProbe:
-          httpGet:
-            path: /foo
-            port: 8001
-          initialDelaySeconds: 5
-          periodSeconds: 5
-{{< /text >}}
-
-This approach allows you to enable the health check prober rewrite gradually on each deployment without reinstalling Istio.
+### Separate port
 
-#### Re-deploy the liveness health check app
+Another alternative is to use separate port for health checking and regular traffic.  Separate port is not recommended as it requires changing your application's health check on a separate port. This option should only be explored when the `probe rewrite` option doesn't work.
```

Comment:
this option doesn't even work today

Reply:
hmm why would separate port won't work?

Reply:
we capture all ports. It would work if you use a separate port AND use excludeInboundPorts I guess, but no one will do that I guess. A readiness port on a different port is useless too.. the whole point is its testing the app, if you test another port it loses value

Reply:
That is basically what the sample does.  It exposes the health check on a separate port than the main app.  We had it as before we had probes rewrite.   Now the reason i didn't delete yet it is if it can be a workaround when probes rewrite is disabled.   @myidpt @frankbu thoughts?